kitsune/users/auth.py

```diff
@@ -51,6 +51,30 @@ def get_token(self, payload):
         self.refresh_token = token_info.get("refresh_token")
         return token_info
 
+    @classmethod
+    def refresh_access_token(cls, refresh_token):
+        """Gets a new access_token by using a refresh_token.
+
+        returns: the actual token or None
```

Comment:
should this be "returns: contents of the token_info endpoint, or an empty dict"?